config/config.yaml

```diff
@@ -110,6 +110,8 @@ Router:
     forceHTTP1: true
     httpTimeout: 120s
     httpMaxIdleConnsPerHost: 32
+  BQSTREAM:
```

Comment:
need to add this config in the operator too if we are planning to only enable transformerProxy for this destination

Reply:
Then how will we enable the transformerProxy for this destination alone ?

Reply:
This file wont be present when deployed to production. To have this variable enabled for a particular destination we need to change it in rudderstack-operator repo